dts/bindings/base/pm.yaml

```diff
@@ -30,3 +30,8 @@ properties:
     description: |
       Automatically configure the device for runtime power management after the
       init function runs.
+
+  zephyr,disabling-power-states:
```

Comment:
why put the zephyr, prefix here?

Reply:
Because this property is Zephyr's specific. https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#the-zephyr-prefix

Reply:
do we share the pm binding with an upstream linux binding?

Reply:
some properties ... `min-residency-us` and `exit-latency-us`

Reply:
is the pm binding in zephyr supposed to be analogous to the `domain-idle-states` linux binding?

Reply:
there are similarities / inspiration. But we have it tailored to Zephyr needs

Reply:
That's why I am wondering why we need the zephyr, prefix here, if the binding itself is zephyr specific

Reply:
nevermind, this is the base pm.yaml, I thought it was a compatible binding, I read wrong.